assets/vendor/js/profile_page.js

```diff
@@ -19,9 +20,59 @@ function saveBio() {
 }
 
 window.onload = function () {
-  var bio = "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.</p>";
-  var x = document.getElementById("def_block");
-  x.innerHTML = bio;
+  console.log("hii");
+  // var bio = "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.</p>";
+  // var x = document.getElementById("def_block");
+  // x.innerHTML = bio;
   var y = document.getElementById("textarea");
   y.innerHTML = bio;
+  $.post("../../backend/profile_page.php", {
```

Comment:
sending a post request but profile_page.php was built for a GET request. For more improved code, U can make the correction in profile_page.php
change $_GET to $_POST